deutsches-archaologisches-institut.csl

```diff
@@ -292,7 +278,7 @@
     </date>
   </macro>
   <macro name="collection-title">
-    <text variable="collection-title"/>
+    <text variable="container-title" strip-periods="true"/>
```

Comment:
this seems unfortunate: a macro that's called collection-title shouldn't just be calling `container-title`. I'd get rid of the macro entirely and just use the `<text variable="container-title" strip-periods="true"/>` wherever you need it.